ts/src/base/Exchange.ts

```diff
@@ -6857,6 +6857,36 @@ export default class Exchange {
         return this.filterBySymbolSinceLimit (sorted, symbol, since, limit) as LongShortRatio[];
     }
 
+    handleTriggerPricesWithPrecision (symbol, params, omitParams = true) {
```

Comment:
I would probably still call it withParams instead, no?

Reply:
ah, i had pushed the changes but forgot to left a comment. so, the PR could be ready, lmk.